settings/Mailing.setting.php

```diff
@@ -91,6 +91,28 @@
     'is_contact' => 0,
     'help_text' => NULL,
   ],
+  'civimail_unsubscribe_methods' => [
+    'group_name' => 'Mailing Preferences',
+    'group' => 'mailing',
+    'name' => 'civimail_unsubscribe_methods',
+    'type' => 'Array',
+    'quick_form_type' => 'Select',
+    'html_type' => 'Select',
+    'html_attributes' => [
+      'multiple' => 1,
+      'class' => 'crm-select2',
+    ],
+    'default' => version_compare(CRM_Utils_System::version(), '5.72', '<=') ? ['mailto'] : ['mailto', 'http', 'oneclick'],
```

Comment:
What changed in CiviCRM 5.72?

Reply:
For purposes of 5.69/5.70, `oneclick` should be an opt-in (per caveats in description; default is just `['mailto']` for continuity).  But ideally, the default *should* change to include the newer methods. This sets a clock / it defines the transition period. We don't have to remember to go patch again in some other/future PR/branch. (Although if someone wants to feel good about themselves, they can trim it down later.)